### ISSUE TITLE
{{searchtag}} Is there a way to hide the category dropdown menu ?

### ISSUE THREAD

**Issue by trebmuh** [dokufreaks/plugin-tag#185]
(from https://github.com/dokufreaks/plugin-pagelist/issues/48 )

I'm a discovering searchtag and didn't find something in the doc (https://www.dokuwiki.org/plugin:tag#searchtags) to achieve this. I was looking at going from here :
![searchtag](https://cloud.githubusercontent.com/assets/8705846/8891216/ccf3512c-3320-11e5-8e2c-7247b825d66c.png)
to something which could looks like:
![searchtag2](https://cloud.githubusercontent.com/assets/8705846/8891219/db1e33c0-3320-11e5-9e67-8e0de323017a.png)

**Comment by LarsGit223** [comment]
Please see the code in my pull-request. Using my version you could write ```{{searchtags&nonsform}}``` to disable the namespace/catégories drop-down box.

**Comment by trebmuh** [comment]
@LarsGit223 I've applied your patch to my current dokuwiki (Frustratric Manner) and it does look to work as expected.

I only had to add a missing "}" at the end of the second lot of your patch to make it showing the page. If not, I was shown with an empty page. I don't know if that's relevant to you, so I decided to share it here anyway.

I've reverted your patch on my current dokuwiki installation since I don't want to mess up with dokuwiki, and I'll wait patiently for an update of this plugin to pop up in my dokuwiki admin panel.

Thanks for that, really appreciated.

**Comment by LarsGit223** [comment]
@trebmuh: could you please re-test the patch by applying the complete file ```syntax/searchtags.php```? I just re-tested and could not see an error regarding the braces.

**Comment by trebmuh** [comment]
@LarsGit223 , I'm sorry that this has been off my radar for so long.
I've been moving from Greebo to Hogfather on the last days, so I'm cleaning up stuff around my dokuwiki installations. Then I found back this very issue.
I've tried to apply the complete file and it works perfectly now, I probably got my brushes mixed up last time.

Thanks you very much for this improvement. Very appreciated in my workflow.
